### DISCUSSION TITLE
Unable to sign in to school account because of bookmarks.

### DISCUSSION THREAD

**Discussion by bananstache** [3kh0/ext-remover#1011] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
ik this is the wrong place to ask for help, but I recently used ltmeet flood method and accidentally put the folder with 2000 in around 3 times. now when I sign into my account, it freezes when loading the bookmarks. I know it's the bookmarks because i managed what i sync on my home computer and turned off bookmarks then it loaded like normal. but on my school chromebook, the admins blocked managing what i sync. and i dont want to ask my sysadmins bc they will get me in trouble for having a buncha exploits bookmarked on my chromebook.

**Comment by Quartinal** [comment, marked as answer]
How much RAM does your Chromebook have? To find how much it has, simply press Ctrl+Search+Escape to open the Diagnostics page on the sign-in screen, and then look at the maximum available memory in the "Memory" tab. 

If you find it has a low amount, you could try to do it on a home PC with more memory (i.e. a Windows 11 PC with 16 gigabytes of RAM). To do this open Chrome and then press the "Add" button under the large "+" and then enter your school account credentials. Then, find your LTMEAT Flood-generated folder. After that, right click on your LTMEAT Flood-generated folder and click "Delete".

Now, try to sign in to your school Chromebook again.
